packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -215,7 +119,7 @@ regen=
 test_list=( "OptimismPortalKontrol.prove_finalizeWithdrawalTransaction_paused" \
```

Comment:
might be useful to document when to update this list in the README.

Reply:
Great point, added a brief "Add New Proofs" section in 1861fde98 which explains this. I plan to write more details here in a future PR